### PR TITLE
Added CinemaHall, MovieSession model, dao and service

### PR DESCRIPTION
**Pull request by aleksandr-hrankin**

### CODE REVIEW

**Review by DmGrt — APPROVED** [review]
It looks like everything is fine

**Review by RomanHreits — APPROVED** [review]
Good job!

File: src/main/java/ua/mate/cinema/model/MovieSession.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+@NoArgsConstructor
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "movie_session_id")
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "movie_id")
```
Comment (RomanDubovskyi) [inline src/main/java/ua/mate/cinema/model/MovieSession.java:26]: is this annotation necessary?

File: src/main/java/ua/mate/cinema/model/MovieSession.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    private Movie movie;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "cinema_hall_id")
```
Comment (RomanDubovskyi) [inline src/main/java/ua/mate/cinema/model/MovieSession.java:29]: same

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**